Task Liverpool/src/main/java/pages/SearchResultsPage.java

```diff
@@ -0,0 +1,152 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SearchResultsPage extends BasePage{
+    private By numberOfResultsFromUpperRightLabel = By.cssSelector(".a-plp-results-title span");
+    private By searchResultsDisplayed = By.cssSelector("li.m-product__card");
+    private By listViewButton = By.cssSelector(".icon-vistalista");
+    private By sortByDropDownList = By.cssSelector(".col-lg-9 #sortby");
+    private By sortByRelevance = By.cssSelector(".col-lg-9 .dropdown-menu [datahref=\"/tienda/Relevancia|0\"]");
+    private By sortByLowPrice = By.cssSelector(".col-lg-9 .dropdown-menu [datahref=\"/tienda/sortPrice|0\"]");
+    private By sortByHighPrice = By.cssSelector(".col-lg-9 .dropdown-menu [datahref=\"/tienda/sortPrice|1\"]");
+    private By sortByRating = By.cssSelector(".col-lg-9 .dropdown-menu [datahref=\"/tienda/rating|1\"]");
+    private By sortByViewed = By.cssSelector(".col-lg-9 .dropdown-menu [datahref=\"/tienda/viewed|1\"]");
+    private By sortBySold = By.cssSelector(".col-lg-9 .dropdown-menu [datahref=\"/tienda/sold|1\"]");
+    private By productPrice = By.cssSelector(".m-product__card .a-card-discount");
+    private By rightArrow = By.cssSelector(".page-link .icon-arrow_right");
+    private By filterLowerThan$500Button = By.id("variants.prices.sortPrice-100-500");
+    //private By filterLowerThan$500Button2 = By.xpath("//input[@id='variants.prices.sortPrice-100-500']"); //Please ignore these locators, I am leaving them here for future reference
+    //private By filterLowerThan$500Button3 = By.cssSelector("#variants\\.prices\\.sortPrice-100-500"); //Please ignore these locators, I am leaving them here for future reference
+    private By noResultsText = By.cssSelector(".a-headline__noResults");
+    private By firstResult;
+    List<WebElement> pricesList;
```

Comment:
what is the propouse of these attributes? I think it not make sence to have it

Reply:
I use them to get the product prices WebElements and then their Integer values, so I can compare and validate that the values are sorted or filtered.